### PR TITLE
【映射文档】对齐与修复了部分映射文档 part 2

### PR DESCRIPTION
**Pull request by RedContritio**
与 PaConvert 进行了映射关系对齐

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.remainder_.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ paddle.Tensor.remainder_(, name=None)
 | divisor         | y            | 除数，PyTorch 可为 Tensor or Scalar，Paddle 仅可为 Tensor，需要转写。   |
 
 ### 转写示例
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.remainder_.md:23]: 上面这个Paddle API的签名都没写全

File: docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft.md [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 ```python
-torch.fft.rfft(input, s=None, dim=(- 2, - 1), norm='backward', *, out=None)
+torch.fft.rfft(input, n=None, dim=(- 2, - 1), norm='backward', *, out=None)
 ```
 
 ### [paddle.fft.rfft](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/fft/rfft_cn.html#rfft)
 
 ```python
-paddle.fft.rfft(x, s=None, axes=(- 2, - 1), norm='backward', name=None)
+paddle.fft.rfft(x, n=None, axes=(- 2, - 1), norm='backward', name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/fft/torch.fft.rfft.md]: infoflow 2024-07-18 21-56-33

这些拼写问题为啥都没有扫出来呢，我比较不确定的是，咱们这次全面清理了，还有多少漏网之鱼

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.relu6.md [hunk 3]
```diff
@@ -17,4 +17,5 @@ paddle.nn.functional.relu6(x, name=None)
 
 | PyTorch       | PaddlePaddle | 备注                                                   |
 | ------------- | ------------ | ------------------------------------------------------ |
-| input           | x           | 表示输入的 Tensor ，仅参数名不一致。               |
+| input         | x           | 表示输入的 Tensor ，仅参数名不一致。               |
+| inplace i     | -            | 表示在不更改变量的内存地址的情况下，直接修改变量的值，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.relu6.md]: inplace i -> inplace

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.remainder_.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ torch.Tensor.remainder_(divisor)
 ### [paddle.Tensor.remainder_](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#id15)
 
 ```python
-paddle.Tensor.remainder_(, name=None)
+paddle.Tensor.remainder_(y, name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.remainder_.md:12]: 这个应该是other还是divisor

infoflow 2024-07-19 11-19-05

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
 
 | PyTorch | PaddlePaddle | 备注                                                |
 | ------- | ------------ | --------------------------------------------------- |
-| input   | input        | 表示输入的 Tensor 。                                |
+| input   | x        | 表示输入的 Tensor 。                                |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md]: 加一句：仅参数名不一致

**Review by zhwesky2010 — APPROVED** [review]

**Comment by zhwesky2010** [comment]
@RedContritio 这个尽快修改一下

**Review by zhwesky2010 — APPROVED** [review]
